compare-metrics-file.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import sys
+import gzip
+
+
+def read_metrics_file(file_name):
+    metric_data = {}
+    with gzip.open(file_name, "rt") if file_name.endswith(".gz") else open(
+        file_name, "r"
+    ) as fin:
+        header = fin.readline()
+        metric_data["header"] = header.split(",")[1:]
+        for line in fin:
+            fields = [x.strip() for x in line.split(",")]
+            metric_data[fields[0]] = fields[1:]
+    return metric_data
+
+
+def compare_metric_files(ref_metrics, query_metrics):
+
+    if ref_metrics["header"] != query_metrics["header"]:
```

Comment:
Do headers ever contain information about the run of the tool that produced them? (i.e. date or timestamp?)

Reply:
Oh are the headers just the column names in this case? (Do the metrics files contain additional headers besides the column names?)

Reply:
headers do not contain any meta information other than the column names